### ISSUE TITLE
Ruby Course: Update Code Block Delimiters

### ISSUE THREAD

**Issue by aconstas** [TheOdinProject/curriculum#27017]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

Codeblocks should be updated to align with new changes in our [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

In the sections named in this issue, codeblocks will be opened by a set of 3 tildes (the ~ character) and then closed by another set of 3 tildes. To align with the style guide, these tilde characters need to change to backticks (the ` character). Be sure to refer to [the codeblocks section](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#codeblocks) of the style guide for an example of what this should look like.
## Guidance:
- Comment on this issue to get a section assigned to you. You can also ask for a specific one
- Please wait until you are assigned by a maintainer before starting to work on this issue to avoid multiple people working on the same sections
- Make sure to read the [contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md) before starting your first contribution
- Please add the following to your PR in the Issues section: Related to #xxxxx, since this issue is divided into sections, individual PRs should NOT INCLUDE the closes keyword.
- **Please ensure that you update the delimiters in every listed lesson of the section you are assigned to.**

### [Advanced Ruby (2 lessons)](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby#advanced-ruby)
- [x] @Servante ***Blocks, Pattern Matching***

### [Basic Ruby (12 lessons)](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby#basic-ruby)
- [x] @riarsdev ***Arrays, Basic Data Types, Basic Enumerable Methods, Conditional Logic, Debugging, Hashes, Input and Output, Loops, Methods, Nested Collections, Predicate Enumerable Methods, Variables***

### [Basic Ruby Projects (4 lessons)](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby#basic-ruby-projects)
- [x] @Torenrob ***Bubble Sort, Caesar Cipher, Stock Picker, Sub Strings***

### [A Bit of Computer Science (3 lessons)](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby#a-bit-of-computer-science)
- [x] @kamranzafar4343 ***Binary Search Trees, Knights Travails, Space Complexity***

### [Files and Serialization (1 lesson)](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby#files-and-serialization)
- [x] @jeremy-gleason ***Project Event Manager***

### [Introduction (1 lesson)](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby#introduction)
- [x] @kamranzafar4343 ***Installing Ruby***

### [Object-Oriented Programming Basics (1 lesson)](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby#object-oriented-programming-basics)
- [x] @kamranzafar4343 ***Object Oriented Programming***

### [Testing Ruby with RSpec (1 lesson)](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby#testing-ruby-with-rspec)
- [x] @kamranzafar4343 ***Introduction to RSpec***

@JoshDevHub

### Path

Ruby / Rails

### Lesson Url

/ruby/

**Comment by riarsdev** [comment]
I would like to contribute. Assign me to work on the `Basic Ruby` section.

**Comment by JoshDevHub** [comment]
Thank you for volunteering @riarsdev 

I've assigned you the Basic Ruby section.

**Comment by Servante** [comment]
I'll take one of these :)

**Comment by JoshDevHub** [comment]
Awesome @Servante 

I've assigned you the Advanced Ruby section

**Comment by Torenrob** [comment]
I'd like to take one of these sections as well.

**Comment by JoshDevHub** [comment]
Thank you for volunteering @Torenrob 

I've assigned you the Basic Ruby Projects section.

**Comment by kamranzafar4343** [comment]
I would also like to contribute. Assign me one of these sections.

**Comment by JoshDevHub** [comment]
You got it @kamranzafar4343 

I've assigned you the "A Bit of Computer Science" section.

**Comment by jeremy-gleason** [comment]
I'd like to contribute also. Can I be assigned to one (or more) of the remaining sections?

**Comment by JoshDevHub** [comment]
@jeremy-gleason Thank you for volunteering.

I've assigned you to the Files and Serialization section.

**Comment by kamranzafar4343** [comment]
Can I be assigned to all of the remaining sections? It would be a great favour.

**Comment by JoshDevHub** [comment]
Given they're all 1 lesson in each section, I can give you the remaining ones @kamranzafar4343 

Thank you for volunteering to finish up the work.

**Comment by JoshDevHub** [comment]
Thanks for contributing everyone!
